.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea/
```

Comment:
This should go in a [global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore), not in the local file.